src/olympia/devhub/templates/devhub/index.html

```diff
@@ -1,5 +1,81 @@
-  {% include "devhub/new-landing/index.html" %}
-{% else %}
-  {% include "devhub/index-legacy.html" %}
-{% endif %}
+{# Our new DevHub landing page represents an entirely different and new design
+   which is why this page doesn't inherit any of our base templates. #}
+<!DOCTYPE html>
+<html lang="{{ LANG }}" dir="{{ DIR }}">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% if not settings.ENGAGE_ROBOTS %}
+      <meta name="robots" content="noindex">
+    {% endif %}
+    {% block extrameta %}{% endblock %}
+
+    <title>{% block title %}{{ dev_page_title() }}{% endblock %}</title>
+
+    <link rel="shortcut icon" type="image/x-icon"
+          href="{{ STATIC_URL }}img/favicon.ico">
+
+    {% block rss_feed %}{% endblock %}
+
+    {# L10n: {0} is an application, like Firefox. #}
+    <link title="{{ _('{0} Add-ons')|f(APP.pretty) }}"
+          rel="search" type="application/opensearchdescription+xml"
+          href="{{ url('amo.opensearch') }}" />
+
+    {% block site_css %}
+      {{ css('devhub/new-landing/css') }}
+    {% endblock %}
+
+    {% block extrahead %}{% endblock %}
+
+    {% if request.user.is_authenticated() %}
+      <meta name="csrf" content="{{ csrf_token }}">
+    {% endif %}
+
+    {% if settings.DEBUG %}
+      {% if settings.LESS_LIVE_REFRESH %}
+        <meta id="live_refresh" name="live_refresh" value="1">
+      {% endif %}
+      {{ js('debug') }}
+    {% endif %}
+
+    {{ js('preload') }}
+  </head>
+  {% set user_authenticated = request.user.is_authenticated() %}
+
+  <body class="html-{{ DIR }} {{ request.APP.short }} {% if user_authenticated %}user-signedin{% endif %} {% block bodyclass %}{% endblock %}"
+        data-raven-dsn="{{ settings.RAVEN_DSN }}"
+        data-raven-urls="{{ settings.RAVEN_ALLOW_LIST }}"
+        {% block bodyattrs %}{% endblock %}>
+
+
+    {% include 'devhub/new-landing/components/navigation.html' %}
+
+    {% if not user_authenticated %}
+      {% include 'devhub/new-landing/components/overview.html' %}
+      {% include 'devhub/new-landing/components/port_chrome_ext.html' %}
+      {% include 'devhub/new-landing/components/publish_your_addon.html' %}
+      {% include 'devhub/new-landing/components/manage_your_addon.html' %}
+    {% endif %}
+
+    {% if not user_authenticated %}
+      <div class="DevHub-MoreInformation scheme-dark content">
+        <div>
+          {% include 'devhub/new-landing/components/blog_posts.html' %}
+          {% include 'devhub/new-landing/components/tools.html' %}
+        </div>
+      </div>
+    {% else %}
+      {% include 'devhub/new-landing/components/my-addons.html' %}
+    {% endif %}
+
+    {% include 'devhub/new-landing/components/footer.html' %}
+    <div id="tooltip">
+        <span></span>
+    </div>
+    {% block site_js %}
+      <script src="{{ CDN_HOST }}{{ cache_buster(url('jsi18n')) }}"></script>
+      {{ js('common') }}
```

Comment:
Out of scope for this patch, but there's a whole mountain of javascript in 'common' - starting a new minimal bundle would be better as it doesn't use any of the existing templates so require anything in those.